### PR TITLE
display new scm revisionData

### PR DESCRIPTION
**Pull request by ghedamat**
see
ember-cli/ember-cli-deploy#305
https://github.com/ember-cli-deploy/ember-cli-deploy-redis/pull/56
https://github.com/ember-cli-deploy/ember-cli-deploy-revision-data/pull/29

NOTE:

to render properly this needs
https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/pull/9

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,100 +20,22 @@ module.exports = {
       },
 
       displayRevisions: function(context) {
-        if(!context.revisions) {
+        if(!context.revisions || context.revisions.length === 0) {
```
Comment (lukemelia) [inline index.js]: We should consider accessing this via `readConfig`

Comment (ghedamat) [inline index.js]: I might still be asleep but I don't think we can do that
https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/blob/master/index.js#L45

**Comment by lukemelia** [comment]
Looks good!

**Comment by ghedamat** [comment]
@lukemelia updated with readme

**Comment by lukemelia** [comment]
:+1:

**Pull request merged**